docs/modules/loader.md

```diff
@@ -0,0 +1 @@
+# loader
```

Comment:
Document the NSO replacement and patching capabilities, noting certain quirks like including `rtld.stub`, `sdk.stub`, etc. files, the ability to leave trailing zeroes off the end of IPS file names, the arrangement of IPS files within `atmosphere/exefs_patches`, and perhaps also a short guide on authoring IPS patches. Ping me over any details you might not be sure about.